### PR TITLE
Small corrections to Belgian, Serbian and Slovenian keymaps

### PR DESCRIPTION
**Pull request by vomindoraan**
## Description

Small corrections to Belgian (#8349), Serbian (#8560) and Slovenian (#8350) keymaps.

Namely, the Belgian keymap calls the non-dead-key degree sign `RNGA` instead of `DEG`, the Slovenian keymap does the opposite for the ring accent dead key, and the Serbian keymaps have a couple of wrong names altogether (one of them causes a multiple definition error).

/request_review @fauxpark 

## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: quantum/keymap_extras/keymap_greek.h [hunk 1]
```diff
@@ -114,7 +114,7 @@
 #define GR_PLUS S(GR_EQL)  // +
 // Row 2
 #define GR_COLN S(GR_SCLN) // :
-#define GR_DTON S(GR_FSIG) // ΅ (dead)
+#define GR_DIAT S(GR_FSIG) // ΅ (dead)
```
Comment (fauxpark) [inline quantum/keymap_extras/keymap_greek.h:117]: Why this change? You suggested `DTON` in #8636.

File: quantum/keymap_extras/keymap_slovenian.h [hunk 2]
```diff
@@ -137,11 +137,11 @@
 #define SI_CARN ALGR(SI_2)    // ˇ (dead)
 #define SI_CIRC ALGR(SI_3)    // ^ (dead)
 #define SI_BREV ALGR(SI_4)    // ˘ (dead)
-#define SI_DEG  ALGR(SI_5)    // ° (dead)
+#define SI_RNGA ALGR(SI_5)    // ° (dead)
```
Comment (fauxpark) [inline quantum/keymap_extras/keymap_slovenian.h:140]: Interesting... the Slovenian layout produces the degree symbol if you cancel the dead key; it is indeed technically a "ring above" symbol due to its deadness... but seems to only combine with "u".

File: quantum/keymap_extras/keymap_greek.h (hunk 1 above)
Comment (vomindoraan) [reply quantum/keymap_extras/keymap_greek.h:117]: I realized that `DIAT` might make more sense since I found out that “tonos” means accent. So this would be similar to how in `RNGA` and `BRKP` the “modifier” word takes up the first three letters. Plus, now I think that `DIAT` pairs nicer with `DIAE` ¯\\\_(ツ)\_/¯

File: quantum/keymap_extras/keymap_slovenian.h (hunk 2 above)
Comment (vomindoraan) [reply quantum/keymap_extras/keymap_slovenian.h:140]: That's correct. Most layouts that have a ring above dead key actually use the degree sign, just flagged for deadness.

Another one that threw me off is the low single quote dead key in the Serbian layout (`RS_LSQU`), which behaves as a cedilla (`c`→`ç`, `s`→`ş`), but when you cancel it you do get the low single quote character. I opted not to change that one, though, since there already is an actual cedilla key in that layout...

Comment (vomindoraan) [reply quantum/keymap_extras/keymap_slovenian.h:140]: I think the `RNGA`s in the Serbian Latin and Croatian layouts also only combine with U to give Ů (used in Czech).

**Comment by drashna** [comment]
Would like to merge this.

**Comment by vomindoraan** [comment]
@fauxpark Done!

**Comment by fauxpark** [comment]
Thanks!
